src/adapters/interlay.ts

```diff
@@ -28,14 +28,14 @@ export const interlayRoutersConfig: Omit<CrossChainRouterConfigs, "from">[] = [
       weightLimit: DEST_WEIGHT,
     },
   },
-  // {
-  //   to: "statemint",
-  //   token: "USDT",
-  //   xcm: {
-  //     fee: { token: "DOT", amount: "1000000000" },
-  //     weightLimit: DEST_WEIGHT,
-  //   },
-  // },
+  {
+    to: "statemint",
+    token: "USDT",
+    xcm: {
+      fee: { token: "DOT", amount: "1000000000" },
```

Comment:
I _think_ this is fine, to be confirmed with Sander

Reply:
It's working for Parallel: https://statemint.stg.subscan.io/xcm_message/polkadot-7e5191d8900b822761e5a8d93bc9f8299aeb4aa9 so for now it should be ok. If statemint increases their fees just like statemine did it might not be enough but idk if that's in the plans, and couldn't find any info on it